### ISSUE TITLE
Brackets in cases environment (amsmath)

### ISSUE THREAD

**Issue by molke-productions** [brucemiller/LaTeXML#1675]
Then there is one more where LaTeXML’s behavior differs from LaTeX.
```
\documentclass{article}

\usepackage{amsmath}

\begin{document}
\[
\begin{cases}
a & x_1\\
[b] & x_2\\[12pt]
[c] & x_3
\end{cases}
\]
\end{document}
```
Somehow, the implementation in amsmath.sty allows both line breaks, but LaTeXML expects (as usual) a length argument after the first one, and so gobbles “[b]” in the second line.

**Comment by dginev** [comment]
Fun! Thanks for debugging all of these, great to have them laid out so nicely.

**Comment by brucemiller** [comment]
Ha! I thought we'd had that one covered... I guess not!
(AMS' & LaTeX' \\ work differently w.r.t skipping whitespace)

**Comment by molke-productions** [comment]
Yes, I thought that there already was some issue regarding this, but didn’t find it.

**Comment by dginev** [comment]
This is what I am seeing with the latest latexml:
![image](https://user-images.githubusercontent.com/348975/136025454-92545fc8-ec4c-4c79-8981-05c0bd92a0c0.png)

Could you share the version of latexml you're running @molke-productions ?

You/anyone can try the latest one here, if convenient (just updated it):
https://latexml.mathweb.org/editor

**Comment by molke-productions** [comment]
Here, I have latexml (LaTeXML version 0.8.5).

**Comment by dginev** [comment]
Installed through a package manager I'd assume? If so, hopefully you'll get a chance to update to 0.8.6 where this specific issue should be fixed.

I found the exact PR where Bruce fixed this (indeed only available in 0.8.6), which is #1458  

So closing here, but thanks for reporting again. It may be best to upgrade before debugging further though, some other bits could get resolved for free.

**Comment by molke-productions** [comment]
Sorry, after updating to the latest version (actually available in Arch Linux), this is indeed fixed.
(I’m a bit reluctant at upgrading LaTeXML because we had to use hacked versions of Image.pm and LaTeXImages.pm for our use case.)
